newrelic-agent/src/main/java/com/newrelic/agent/config/JfrConfigImpl.java

```diff
@@ -18,12 +20,19 @@
     public static final String AUDIT_LOGGING = "audit_logging";
     public static final Boolean AUDIT_LOGGING_DEFAULT = Boolean.FALSE;
     public static final Boolean USE_LICENSE_KEY_DEFAULT = Boolean.TRUE;
+    public static final String HARVEST_INTERVAL = "harvest_interval";
+    public static final String QUEUE_SIZE = "queue_size";
```

Comment:
If the intent is for these to be publicly configurable then we'll probably want them in the default yaml: https://github.com/newrelic/newrelic-java-agent/blob/main/newrelic-agent/src/main/resources/newrelic.yml#L274-L289

And also in the usage section of the readme: https://github.com/newrelic/newrelic-jfr-core#usage-for-new-relic-java-agent-jfr-service

Thats said, I'm not sure if the intent is for these to be publicized as of yet...